.travis.yml

```diff
@@ -63,7 +65,7 @@ before_install:
   - docker-compose -f docker-compose-ci.yml up -d
 
 install:
-  - if [ "$TEST_SUITE" = "install_stability_1" ] || [ "$TEST_SUITE" = "install_stability_2" ] || [ "$TEST_SUITE" = "install_stability_3-4-5" ]; then docker exec -i social_ci_web bash /var/www/scripts/social/install/install_script.sh; fi
+  - if [ "$TEST_SUITE" = "install_stability_1" ] || [ "$TEST_SUITE" = "install_stability_2" ] || || [ "$TEST_SUITE" = "install_stability_3" ] || || [ "$TEST_SUITE" = "install_stability_4" ] || [ "$TEST_SUITE" = "install_stability_5" ]; then docker exec -i social_ci_web bash /var/www/scripts/social/install/install_script.sh; fi
```

Comment:
I think you have too many double pipes here.

Reply:
@jochemvn thanks for eye open 👍